### PR TITLE
[doc][ybm][DOC-485] GCS and Dynatrace integration

### PR DESCRIPTION
**Pull request by ddhodge**
GCS integration
CLOUDGA-24213
DOC-532

@netlify /preview/yugabyte-cloud/cloud-monitor/managed-integrations/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md [hunk 1]
```diff
@@ -172,6 +175,23 @@ To create an export configuration, do the following:
     http://<victoria-metrics-endpoint-host-address>/opentelemetry
     ```
 
+1. Click **Create Configuration**.
+
+  {{% /tab %}}
+
+  {{% tab header="GCS" lang="gcs" %}}
```
Comment (atharva-yb) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:182]: We mention `Google Cloud Storage` everywhere in UI. So better to keep the same in docs as well?

Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:182]: Well the tabs were getting a bit wide on the screen

Comment (sunilsutarDCE) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:182]: Can we change in UI - **_Google Cloud Storage (GCS)_** ?

Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:182]: No no, in docs I will add "(GCS)" at first occurrance - this is standard for documentation

File: docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md [hunk 2]
```diff
@@ -172,19 +183,41 @@ To create an export configuration, do the following:
...
+
+The GCS integration requires the following:
```
Comment (atharva-yb) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:192]:
```suggestion
The Google Cloud Storage (GCS) integration requires the following:
```
I think we can add full form in description.

**Review by atharva-yb — APPROVED** [review]
Thanks for docs changes.

File: docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md [hunk 3]
```diff
@@ -12,22 +12,31 @@ menu:
...
 
-For information on how to export metrics and logs from a cluster using an integration, refer to [Export metrics](../metrics-export/) and [Export logs](../logging-export/).
+Currently, you can export data to the following tools.
+
+| Integration | Log export | Metric export |
+| :---------- | :--------- | :------------ |
+| [Datadog](https://docs.datadoghq.com/) | Database query logs<br>Database audit logs | [Yes](https://docs.datadoghq.com/integrations/yugabytedb_managed/#data-collected) |
+| [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) | | [Yes](https://grafana.com/grafana/dashboards/12620-yugabytedb/) |
+| [Sumo Logic](https://www.sumologic.com) | | Yes |
+| [Prometheus](https://prometheus.io/docs/introduction/overview/) {{<badge/tp>}} | | Yes |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:37]:
```suggestion
| [Prometheus](https://prometheus.io/docs/introduction/overview/) {{<tags/feature/tp>}} | | Yes |
```

File: docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md [hunk 4]
```diff
@@ -12,22 +12,31 @@ menu:
...
+| [VictoriaMetrics](https://docs.victoriametrics.com/) {{<badge/tp>}} | | Yes |
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:38]:
```suggestion
| [VictoriaMetrics](https://docs.victoriametrics.com/) {{<tags/feature/tp>}} | | Yes |
```

**Review by aishwarya24 — COMMENTED** [review]
make the suggested changes and then it should build fine.

**Review by Aastha-yb — APPROVED** [review]

File: docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md [hunk 5]
```diff
@@ -92,40 +109,43 @@ The Sumo Logic integration requires the following:
...
+
+    The DNS for the endpoint must be in a publicly accessible DNS record, allowing it to resolve globally. This typically involves adding the URL to a public DNS zone. For example, in AWS, this would mean adding the URL to a Public Hosted Zone in Route 53. To confirm that the address is publicly resolvable, you can use a tool like nslookup.
+
   - VPC hosting the Prometheus instance has the following Inbound Security Group rules:
-    - Allow HTTP inbound traffic on port 80 for Prometheus endpoint URL (HTTP)
-    - Allow HTTPS inbound traffic on port 443 for Prometheus endpoint URL (HTTPS)
+    - Allow HTTP inbound traffic on port 80 for Prometheus endpoint URL (HTTP).
+    - Allow HTTPS inbound traffic on port 443 for Prometheus endpoint URL (HTTPS).
 
     See [Control traffic to your AWS resources using security groups](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-security-groups.html) in the AWS documentation.
```
Comment (bansal01yash) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:136]: Need something similar here for GCP, perhaps linking here https://cloud.google.com/firewall/docs/firewalls. cc: @atharva-yb

**Pull request merged**